tests/test_nplinker_local.py

```diff
@@ -42,9 +42,28 @@ def npl() -> NPLinker:
     return npl
 
 
+# ---------------------------------------------------------------------------------------------------
+# After manually checking data files for PODP MSV000079284, we have the following numbers:
+# 370 antismash BGCs
+# 114 GCFs
+#   - 49 singleton GCFs
+#   - 1 mibig-only GCF (not singleton)
+#   - 12 GCFs having mibig bgcs (not singleton, not mibig-only); in total 20 mibig bgcs used in GCFs
+# 25935 spectra
+#   - 24652 spectra have strain info (from strain mapping file)
+#   - 1283 spectra do not have strain info
+# 25769 molecular families
+#   - 25740 singleton families
+#   - 29 non-singleton families
+# 26 strains from strain mapping file
+#   - all strains loaded = 26 (from strain mappings) + 20 (from mibig) = 46
+# ---------------------------------------------------------------------------------------------------
+
+
 @pytest.mark.skipif(os.environ.get("CI") == "true", reason="Skip when running on CI")
 def test_load_data(npl: NPLinker):
     assert len(npl.bgcs) == 390
-    assert len(npl.gcfs) == 113
-    assert len(npl.spectra) == 25935
-    assert len(npl.molfams) == 25769
+    assert len(npl.gcfs) == 64
```

Comment:
As for BGCs, it would be useful to motivate this number (64) in the comments above